packages/e2e/test/web-extension/extension/background/cip30.ts

```diff
@@ -3,7 +3,7 @@ import { cip30 } from '@cardano-sdk/web-extension';
 import { logger, walletName } from '../util';
 import { runtime } from 'webextension-polyfill';
 import { cip30 as walletCip30 } from '@cardano-sdk/wallet';
-import { walletReady } from './wallet';
+import { walletReady } from './walletManager';
 
```

Comment:
Switching the wallet also needs to switch the wallet for cip30
```ts
walletCip30.createWalletApi(walletManagerWorker.wallet$, ...)
```

Reply:
Fixed here: [5f05e62](https://github.com/input-output-hk/cardano-js-sdk/pull/502/commits/5f05e625fce3e754df58a7dc75bdac59d534364b)